t/125_shared_boolean.t

```diff
@@ -5,14 +5,20 @@ use Cpanel::JSON::XS ();
 
 BEGIN {
   plan skip_all => 'no threads' if !$Config{usethreads};
-  eval "require threads::shared 1.21;";
-  plan skip_all => 'no shared_clone' if $@;
-  plan tests => 8;
 }
 
 use threads;
 use threads::shared;
 
+BEGIN {
+  if (eval {threads::shared->VERSION(1.21)}) {
```

Comment:
```suggestion
  if (eval {threads::shared->VERSION('1.21')}) {
```

Reply:
I used `1.21` instead of `'1.21'` on purpose as [perldoc -f use](https://perldoc.perl.org/functions/use) specifically uses numeric rather than the string argument. When I assign to `$VERSION`, I stick with string or `v` values, so it might be preferable to that, but I did not want to go against the docs.

Reply:
use cannot take strings. In all other cases, versions are strings and should be defined as such.

Reply:
This allows, for example, the error message to retain trailing zeroes if they exist, and version.pm to parse it into the integers 1 and 21 without the possibility of floating point errors.

Reply:
I know the benefits of using string values, thank you very much. I was not aware that `use` did not accept string versions. Either way, I think it is more important that the check is valid Perl in the first place.

Pushed the suggested correction.